docs/update_petro_real.rst

```diff
@@ -0,0 +1,185 @@
+rms.update_petro_real
+=======================
+
+When running FMU project where field parameters for both facies and
+petrophysical properties are updated in ERT simultaneously,
+some adjustments are needed in the RMS project to support this type
+of workflow. It is necessary to have one petrosim job per facies.
+
+The module *update_petro_real* will combine values from realization
+of a petrophysical property for individual facies into one common
+petrophysical realization by using the facies realization as filter.
+
+Example describing the workflow
+--------------------------------
+
+The zone Valysar has four different facies::
+
+    Floodplain
+    Channel
+    Crevasse
+    Coal
+
+A petrophysical realization is made for PHIT and KLOGH for each of the three
+first facies with property names::
+
+    Floodplain_PHIT
+    Floodplain_KLOGH
+    Channel_PHIT
+    Channel_KLOGH
+    Crevasse_PHIT
+    Crevasse_KLOGH
+
+Neither PHIT nor KLOGH are updated as field parameters for facies
+Coal since they have constant (not spatially varying) values and
+may even have 0 specified uncertainty.
+The variables VSH and VPHYL are chosen not to be used as field parameters
+in ERT in assisted histroy matching and therefore not included here,
```

Comment:
typo: "histroy". 

"The variables VSH and VPHYL are chosen not to be used as field parameters..." I will guess that is not general but just in this example?

Reply:
yes , fixed the text